tests/ounit_tests/ounit_json_tests.ml

```diff
@@ -1,153 +1,14 @@
 let ( >:: ), ( >::: ) = OUnit.(( >:: ), ( >::: ))
-type t = Ext_json_noloc.t
```

Comment:
The existing `Ext_json` tests were moved to `tests/ounit_tests/ounit_ext_json_tests.ml`